onnxruntime/test/testdata/matmul_integer_to_float.py

```diff
@@ -64,10 +64,18 @@ def GenerateModel(model_name, sign_i, sign_w, has_zp=True, bias=False):  # noqa:
 
 
 if __name__ == "__main__":
-    GenerateModel("matmul_integer_to_float_int8.onnx", False, True)
-    GenerateModel("matmul_integer_to_float_uint8.onnx", False, False)
-    GenerateModel("matmul_integer_to_float_int8_bias.onnx", False, True, False, True)
-    GenerateModel("matmul_integer_to_float_uint8_bias.onnx", False, False, False, True)
+    GenerateModel("matmul_integer_to_float16_int8.onnx", False, True, True)
+    GenerateModel("matmul_integer_to_float16_uint8.onnx", False, False, True)
+    GenerateModel("matmul_integer_to_float16_int8_bias.onnx", False, True, True, False, True)
+    GenerateModel("matmul_integer_to_float16_uint8_bias.onnx", False, False, True, False, True)
```

Comment:
>GenerateModel("matmul_integer_to_float16_uint8_bias.onnx", False, False, True, False, True) [](http://example.com/codeflow?start=4&length=91)

That's a lot of booleans :b. I wish C++ support named parameters for readability...

GenerateModel("matmul_integer_to_float16_uint8_bias.onnx", .model_name = False, .sign_i = False, .sign_w = True, .output_type_fp16 = False, .has_zp=False, .bias=True)

Reply:
updated in next rev